docs/docs/forms.md

```diff
@@ -174,6 +174,132 @@ class FlavorForm extends React.Component {
 
 Overall, this makes it so that `<input type="text">`, `<textarea>`, and `<select>` all work very similarly - they all accept a `value` attribute that you can use to implement a controlled component.
 
+## Multi select
+
+In HTML, you can change a `select` tag to multiselect, with `multiple` attribute:
+
+```html
+<select multiple>
+  <option value="grapefruit">Grapefruit</option>
+  <option value="lime">Lime</option>
+  <option value="coconut">Coconut</option>
+  <option value="mango">Mango</option>
+</select>
+```
+
+You can use this feature on React too:
+
+```js{1}
+<select multiple={true} value={this.state.value}>
+  <option value="grapefruit">Grapefruit</option>
+  <option value="lime">Lime</option>
+  <option value="coconut">Coconut</option>
+  <option value="mango">Mango</option>
+</select>
+```
+
+Becareful, as your select tag is multiple, the value **must be an array**:
+
+```js{3}
+constructor(props) {
+  super(props);
+  this.state = { value: ['lime', 'mango'] };
+}
+```
+
+Now we have a controlled component, so we have to set an `onChange` handler for it:
+
+```html
+<select multiple={true} value={this.state.value} onChange={this.handleChange}>
+```
+
+Writing the `onChange` handler to get current `selected` options is a bit harder
+than the previous ones, we have to loop through the `options`, and get the value
+of selected ones:
+
+```js{3-8}
+  handleChange(event) {
+    var options = event.target.options;
+    var value = [];
+    for (var i = 0, l = options.length; i < l; i++) {
+      if (options[i].selected) {
+        value.push(options[i].value);
+      }
+    }
+    
+    this.setState({ value: value });
+  }
+```
+[Try it on CodePen.](http://codepen.io/dashtinejad/pen/yVaojJ?editors=0010)
+
+## Checkbox
+Another form control which you can change it to a controlled component, is `checkbox`:
+
+```html
+<input type="checkbox" />
+```
+
+You can change this checkbox to controlled component by setting the `checked` property,
```

Comment:
I think it would be better to combine the JSX and javascript code in this example. Look at how textarea works, the format is just:

--
In HTML a textarea is like this:

(example)

In React, you change X and Y, so a full example is this:

(full example)

Note some brief stuff stuff.
-- 

How about using that parallel format for checkbox and radio buttons?